modules/ROOT/pages/linking.adoc

```diff
@@ -135,6 +134,61 @@ ordersByOrderId (orderId: “123”) {
 }
 --
 
+== Creating a Link Between Object Types Using Composite Keys
```

Comment:
What does "using" modify? Do you create the link by using the keys, or do the types use the keys?